docs/users/metadata.mdx

```diff
@@ -289,7 +288,7 @@ There are multiple ways to retrieve public metadata.
 
 On the frontend, it's available on the [`User`](/docs/references/javascript/user) object which can be accessed using the [`useUser()`](/docs/hooks/use-user) hook.
 
-On the backend, it's available on the [Backend `User`](/docs/references/backend/types/backend-user) object which can be accessed using the JavaScript Backend SDK's [`getUser()`](/docs/references/backend/user/get-user) method. It can also be attached to a session token, and the `sessionClaims` of the session token can be retrieved on the [`Auth`](/docs/references/backend/types/auth-object) object. If you need to retrieve public metadata frequently in the backend, the best option is to attach it to the session token and retrieve it from the session token. See the [guide on customizing your session token](/docs/backend-requests/custom-session-token).
+On the backend, it's available on the [Backend `User`](/docs/references/backend/types/backend-user) object which can be accessed using the JavaScript Backend SDK's [`getUser()`](/docs/references/backend/user/get-user) method. However, this makes a network request, which is subject to [rate limits](/docs/backend-requests/resources/rate-limits). To avoid a network request, you can attach the metadata as a claim in the user's session token. Then, the metadata can be retrieved from the `sessionClaims` on the [`Auth`](/docs/references/backend/types/auth-object) object. See the [guide on customizing your session token](/docs/backend-requests/custom-session-token).
```

Comment:
```suggestion
On the backend, it's available on the [Backend `User`](/docs/references/backend/types/backend-user) object which can be accessed using the JavaScript Backend SDK's [`getUser()`](/docs/references/backend/user/get-user) method. However, this makes an API request, which is subject to [rate limits](/docs/backend-requests/resources/rate-limits). To avoid a network request, you can attach the metadata as a claim in the user's session token. Then, the metadata can be retrieved from the `sessionClaims` on the [`Auth`](/docs/references/backend/types/auth-object) object. See the [guide on customizing your session token](/docs/backend-requests/custom-session-token).
```